CHANGELOG.md

```diff
@@ -23,6 +23,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/virtual-users/compare/release-3.6.6...master
 
+### Added
+- Add custom logging configuration [JPERF-487].
```

Comment:
Could you please stick to the existing convention?
Example:
>Daemonize WebDriver closing threads. Fix JPERF-470.

So it should be something like:

> Add custom logging configuration. Resolve [JPERF-487].

Reply:
Sure